### ISSUE TITLE
Publish status deprecated in yangdocs

### ISSUE THREAD

**Issue by rolandphung** [openconfig/public#1059]
It seems that the yangdoc documentation does not publish `status deprecated`. This would be a very useful enhancement to be able to read through to docs and know at a glance which of the similar models are the "current" ones.

Example:
https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-routing-policy.html#routing-policy-policy-definitions-policy-definition-statements-statement-actions-bgp-actions-set-ext-community-reference-config-ext-community-set-ref

https://github.com/openconfig/public/blob/master/release/models/bgp/openconfig-bgp-policy.yang#L1169

**Comment by dplore** [comment]
Agreed! I've had the same thought as well.  Will schedule adding this

**Comment by rolandphung** [comment]
Bumping to keep this issue from being stale.
